conf/gulp-tasks/library.gulpfile.js

```diff
@@ -111,11 +113,34 @@ async function createBundleTaskFactory (locale) {
 }
 
 function compileCSS () {
+  return new Promise(resolve => {
+    return parallel(compileLTRCSS, compileRTLCSS)(resolve);
+  });
+}
+
+/**
+ * Compiles the standard left to right CSS
+ */
+function compileLTRCSS () {
+  return src('./src/ui/sass/**/*.scss')
+    .pipe(sass({
+      outputStyle: 'compressed'
+    }).on('error', sass.logError))
+    .pipe(postcss())
+    .pipe(dest('./dist/'));
+}
+
+/**
+ * Compiles the right to left CSS which is used for languages such as Arabic
+ */
+function compileRTLCSS () {
   return src('./src/ui/sass/**/*.scss')
     .pipe(sass({
       outputStyle: 'compressed'
     }).on('error', sass.logError))
+    .pipe(rtlcss())
```

Comment:
(github won't let me start a new conversation on the same line as an already resolved one)
is there an easy way to pass in the file name? in webpack it would be something like `[filename].rtx.[ext]`

Reply:
yep, updated to append '.rtl' to the basename